Patch to enable support for slusb module.
By: Marvin Stodolsky <marvin.stodolsky@gmail.com> from an un-released version of sl-modem.
Index: sl-modem-2.9.11~20080817/drivers/Makefile
===================================================================
--- sl-modem-2.9.11~20080817.orig/drivers/Makefile	2009-03-02 07:40:18.000000000 +0200
+++ sl-modem-2.9.11~20080817/drivers/Makefile	2009-03-02 07:52:43.000000000 +0200
@@ -19,7 +19,10 @@
 KERNEL_DIR:=/lib/modules/$(shell uname -r)/build
 SUBLEVEL:=$(shell echo $(KERNEL_DIR) | cut -d. -f3 | cut -d. -f1 | cut -d-  -f1)
 # slusb support ends at 2.6.24
+
+ifndef USB
 AGE:=$(shell if [ "$(SUBLEVEL)" -gt 24 ] ;then echo NEW;fi)
+endif
 
 FOUND_CLASS_SIMPLE := $(shell grep -q 'class_simple_device_add' ${KERNEL_DIR}/include/linux/device.h 2> /dev/null && echo -DFOUND_CLASS_SIMPLE)
 
@@ -59,7 +62,7 @@
 
 dep:
 clean:
-	$(RM) kernel-ver $(obj-m) $(obj-m:.o=.ko) *st7554.o amrmo_init.o sysdep_amr.o *.mod.* .*.cmd *~
+	$(RM) kernel-ver $(obj-m) $(obj-m:.o=.ko) *st7554.o amrmo_init.o sysdep_amr.o *.mod.* .*.cmd *~ slusb.*
 	$(RM) -r .tmp_versions
 
 install-devices:
